tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
     "lib": ["es6", "dom", "es2017"],                /* Specify library files to be included in the compilation. */
     "allowJs": true,                                /* Allow javascript files to be compiled. */
-    "types": ["node", "jquery", "./lib/jquery.jsonp-2.4.min", "cypress", "@testing-library/cypress"],
+    "types": ["node", "jquery", "jqueryui", "./lib/jquery.jsonp-2.4.min", "cypress", "@testing-library/cypress"],
```

Comment:
@mpeyper you were right that this would get out of hand fast :p

Reply:
Yeah, but I'm not sure if there's an alternative way